index.html

```diff
@@ -504,71 +475,72 @@ <h3>
           </p>
           <ul>
             <li>
-              <dfn>"navigation"</dfn>, if the request is a [=navigation
+              <code>"navigation"</code>, if the request is a [=navigation
               request=];
             </li>
             <li>
-              <dfn>"css"</dfn>, if the request is a result of processing a CSS
-              <a data-xref-type="css-function">url()</a> directive such as
+              <code>"css"</code>, if the request is a result of processing a CSS
+              <a data-cite="css-values-4"
+              data-xref-type="css-function">url()</a> directive such as
               <code>@import url()</code> or <code>background: url()</code>;
               [[CSS-VALUES]]
             </li>
             <li>
-              <dfn>"script"</dfn>, if the request is a result of loading any
+              <code>"script"</code>, if the request is a result of loading any
               <a data-cite="HTML#concept-script">script</a> (a classic
```

Comment:
ditto exported?

Reply:
It's exported, but respec can't deal with `[=script=]`. We can deal with it as a followup.
/cc @marcoscaceres